R/example_data.R

```diff
@@ -0,0 +1,57 @@
+# https://github.com/2DegreesInvesting/tiltIndicator/blob/main/R/example_dictionary.R#L128
+example_data_factory <- function(data) {
+  force(data)
+
+  function(...) {
+    new <- tibble(...)
+    old <- as.list(data)
+    add <- setdiff(names(old), names(new))
+    as_tibble(append(new, old[add]))
+  }
+}
```

Comment:
@lindadelacombaz this function comes from tiltIndicator but it seems useful everywhere so I may eventually share it in some more reusable way.